mara_minimal_trajectory_action/src/mara_minimal_trajectory_action.cpp

```diff
@@ -1,151 +1,167 @@
-#include <inttypes.h>
-#include <memory>
-
-// ROS 2.0
 #include "rclcpp/rclcpp.hpp"
 #include "rclcpp_action/rclcpp_action.hpp"
-#include "rcutils/cmdline_parser.h"
-
-// HRIM
 #include "hrim_actuator_rotaryservo_actions/action/goal_joint_trajectory.hpp"
 
-rclcpp::Node::SharedPtr g_node = nullptr;
+#include <inttypes.h>
+#include <memory>
 
-void print_usage()
-{
-  printf("Usage for gripper_minimal_client app::\n");
-  printf("ros2 run mara_minimal_trajectory_action mara_minimal_trajectory_action [-t topic] [-h]\n");
-  printf("options:\n");
-  printf("-h : Print this help function.\n");
-  printf("-t action name : Name of the action.\n");
-}
+using HRIMJointTrajectory = hrim_actuator_rotaryservo_actions::action::GoalJointTrajectory;
+using GoalHandleHRIMJointTrajectory = rclcpp_action::ClientGoalHandle<HRIMJointTrajectory>;
+
+// node as global variable
+rclcpp::Node::SharedPtr node = nullptr;
 
 void feedback_callback(
-  rclcpp_action::ClientGoalHandle<hrim_actuator_rotaryservo_actions::action::GoalJointTrajectory>::SharedPtr,
-  const std::shared_ptr<const hrim_actuator_rotaryservo_actions::action::GoalJointTrajectory::Feedback> feedback)
+  GoalHandleHRIMJointTrajectory::SharedPtr,
+  const std::shared_ptr<const HRIMJointTrajectory::Feedback> feedback)
 {
+  //
+  // ----------- FEEDBACK ----------- //
+  //
+
   double time_from_start = (double)(feedback->actual.time_from_start.sec) +
                            (double)(feedback->actual.time_from_start.nanosec/1e+9);
-  double time_from_start_desired = (double)(feedback->desired.time_from_start.sec) +
-                                   (double)(feedback->desired.time_from_start.nanosec/1e+9);
-  double time_from_start_error = (double)(feedback->error.time_from_start.sec) +
-                                 (double)(feedback->error.time_from_start.nanosec/1e+9);
-
-  RCLCPP_INFO(g_node->get_logger(), "Current degrees: %.2f\ttime:%.5f",
-                                        feedback->actual.positions[0]*180/3.1416,
-                                        time_from_start);
-  RCLCPP_INFO(g_node->get_logger(), "Desired degrees: %.2f\ttime:%.5f",
-                                        feedback->desired.positions[0]*180/3.1416,
-                                        time_from_start_desired);
-  RCLCPP_INFO(g_node->get_logger(), "Error degrees: %.2f\ttime:%.5f",
-                                        feedback->error.positions[0]*180/3.1416,
-                                        time_from_start_error);
-  RCLCPP_INFO(g_node->get_logger(), "-----------------------------");
+
+  RCLCPP_INFO(node->get_logger(), "Current degrees: %.2f\ttime:%.5f",
+                                  (double)feedback->actual.positions[0]*180/3.1416,
+                                  time_from_start);
+
+  // TODO: Not working
```

Comment:
please decide if you are going to print it or not. If no please remove the comments